### PR TITLE
Update to latest Equinox OSGI R7 implementations

### PR DESCRIPTION
**Pull request by tjwatson**
For issue #6051

### CODE REVIEW

**Comment by tjwatson** [comment]
Build with all changes necessary in open-liberty and commercial-liberty at:

https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_6LJuQAIUEemkvu7GUlsKfw

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
#build

File: dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops [hunk 1]
```diff
@@ -30,15 +30,19 @@
...
 
 # OSGI Log Message
-OSGI_MSG001={0} {1} {2} {3} {4}
+OSGI_MSG001=CWWKE0703I: {0} {1} {2} {3}
 OSGI_MSG001.explanation=An informational message was logged using the OSGi log service
 OSGI_MSG001.useraction=No action is required.
 
-OSGI_WARNING_MSG=CWWKE0700W: {0} {1} {2} {3} {4}
+OSGI_AUDIT_MSG=CWWKE0704A: {0} {1} {2} {3}
+OSGI_AUDIT_MSG.explanation=An audit message was logged using the OSGi log service
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops]: Add a period at the end of the sentence.

File: dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops [hunk 2]
```diff
@@ -30,15 +30,19 @@
 # -------------------------------------------------------------------------------------------------
 
 # OSGI Log Message
-OSGI_MSG001={0} {1} {2} {3} {4}
+OSGI_MSG001=CWWKE0703I: {0} {1} {2} {3}
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops]: This is a global comment: What are the variables for {0} {1} {2} {3} {4}? Are they complete sentences, and is it enough information for a user to determine an action, as the useraction section of the message suggests?

Comment (tjwatson) [reply dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops]: I do not really like the approach here, but I also didn't have a good alternative suggestion so just kept with the current design.  These variables are only used as place holders to file in different components from an OSGi LogEntry.  The reason the nlsprops is used here is to provide the message code so a user can look it up in the documentation for the explanation and user action.

As to how good the information is that the user gets, that all depends on what another bundle logs to the OSGi LogService.  We cannot really control that here in this code.

File: dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops (hunk 1 above)
Comment (tjwatson) [reply dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops]: I'll fix that in a follow on commit.  Looks like I need to put a period at the end of the existing OSGI_MSG001.explanation text also (which is where I copied this message from.

Comment (tjwatson) [reply dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops]: This is now fixed with PR #6119

**Pull request merged**